src/main/io/osd.h

```diff
@@ -95,6 +95,7 @@ typedef enum {
     OSD_ADJUSTMENT_RANGE,
     OSD_CORE_TEMPERATURE,
     OSD_ANTI_GRAVITY,
+    OSD_LOG_STATUS,
```

Comment:
maybe `OSD_BBLOG_STATUS` ?

Reply:
I always prefer the shorter name when equally descriptive. Calling it `BBLOG` would make sense in case another type of logging will be added sometime later. Right now the only type of logging is blackbox logging as far as I know. I'm very open to discussions about naming, just not sure about the benefits of renaming this. As long as it does not lead to confusion I prefer less typing. :)